yt_dlp/extractor/weyyak.py

```diff
@@ -0,0 +1,50 @@
+import re
+
+from .common import InfoExtractor
+from ..utils import traverse_obj
+
+
+class WeyyakIE(InfoExtractor):
+    _VALID_URL = r'https?://weyyak\.com/(?P<lang>\w+)/player/(?P<type>episode|movie)/(?P<id>\d+)(?:/.+)?'
+    _TESTS = [
+        {
+            'url': 'https://weyyak.com/en/player/episode/1341952/Ribat-Al-Hob-Episode49',
+            'md5': '0caf55c1a615531c8fe60f146ae46849',
+            'info_dict': {
+                'id': '1341952',
+                'ext': 'm3u8',
```

Comment:
Wrong. Should be mp4/ts. See HobbyistDev's comment

Reply:
It doesn't have mp4 ext, I tried but it says

`[{'format_id': '', 'format_index': None, 'url': 'https://nodea-weyyak.akamaized.net/vod02/WS01893.mp4/master-v1a1.m3u8?em=-pQOsgo9yDyFhM3NfZS0yg&ex=1685151387', 'ext': None, 'protocol': 'm3u8_native', 'preference': None, 'quality': None, 'has_drm': None}]`